docs/UserGuide.md

```diff
@@ -674,6 +675,12 @@ Shows the storage command, and an example of how to use it.
    Example: storage /clear
 ```
 
+### 3.7. Exit application `bye`
+Exits the application. Rest assured that your data have been saved by the application.
+
+##### Format: `bye`
+##### Expected Outcome: `bye` and program terminates.
+
```

Comment:
Expected outcome will be different right?

Reply:
I tried running bye and the program outputs 'bye' and exits. Is there another message it is supposed to print?

Reply:
So I think should put program terminates on another line

Reply:
Fixed.